src/main/java/org/usfirst/frc/team1294/robot/subsystems/ClimbingSubsystem.java

```diff
@@ -1,18 +1,23 @@
 package org.usfirst.frc.team1294.robot.subsystems;
-
+import com.ctre.CANTalon;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import org.usfirst.frc.team1294.robot.RobotMap;
+import org.usfirst.frc.team1294.robot.commands.ClimbRope;
 import edu.wpi.first.wpilibj.command.Subsystem;
 
 /**
  * A subsystem that encompases every motor and sensor related to the rope climbing game mech.
  */
 public class ClimbingSubsystem extends Subsystem {
+	public final CANTalon climbTalon;
 
     public ClimbingSubsystem() {
         super("ClimbingSubsystem");
+        climbTalon = new CANTalon(RobotMap.DRIVEBASE_CLIMBING_TALON);
     }
 
     @Override
     protected void initDefaultCommand() {
-
+    	setDefaultCommand(new ClimbRope());
```

Comment:
remove this.... the command only gets triggered by the button that needs to be assigned in OI